pkgs/applications/networking/p2p/flood/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, buildNpmPackage, fetchFromGitHub }:
+
+buildNpmPackage rec {
+  pname = "flood";
+  version = "4.7.0";
```

Comment:
I tried to backport this commit but it fails to apply cleanly in the current version. We can probably backport more patches to make this work, but it seems that it is better to just override this locally if you need to work with modern qBittorrent versions for now.

Reply:
Another option would be to just package unstable for now, WDYT @winterqt ?

Reply:
BTW, this release is from 2021, so ancient times in JS community. Probably have some security issues already.

So maybe should we go with unstable?

Reply:
Well, I tried to build the latest [commit](https://github.com/jesec/flood/commit/2b652f8148dab7134eeeb201b9d81dd6b8bda074), and while it builds successfully I just got a blank screen and some errors in the developer console.

So I don't know, at least version 4.7.0 seems to works fine.

Reply:
Pushed the bump in a separate commit, so if you want to bump the version to the current development version just say so and we can commit it instead of 4.7.0.